### DISCUSSION TITLE
Add autocomplete to collection and field select

### DISCUSSION THREAD

**Discussion by hanneskuettner** [directus/directus#18022] [Feature Requests]
When creating fields or configuring relations and setting values for eg. Related collections or sort fields I find myself kind of knowing what a collection or field is named.
But right now I either have to remember its full name or scroll through a long list of collections or fields.

I'd really like there to be an autocomplete menu that is filtered by what I already typed to speed up the selection process.

Just a small quality of life improvement.

**Comment by hanneskuettner** [comment]
I've created a PR #18023 that implements this improvement.

**Comment by connorwinston** [comment]
This sounds like a good improvement, however, it seems like this could be something improved across the board in all collection selection interfaces. Maybe we can re-use the search capabilities in `v-select` https://github.com/directus/directus/pull/13867 to make this happen! Would you mind opening a new feature request using the RFC format to address collection selection across the data studio app?

---

This request will now be closed to keep our discussions tidy as it is over 90 days and did not reach 15 upvotes. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
